meetings/2024-07-23.md

```diff
@@ -0,0 +1,32 @@
+# TC Meeting 2024-07-23
+
+Present: Cornelius, Stephanie, Florian, Adrian, Frederik, Max; Excused: Peter, Loic
+
+## Agenda
+
+* Onboarding liblrs, meet the maintainer, next steps
+* DCO bot for TC repo switched on
+* GitHub gave us teams plan for nonprofit organization
+* Managing OpenRail Association owned DNS entries: https://github.com/OpenRailAssociation/openrail-dns/
+* Discussion: For describing projects, would it make sense to adopt something like https://github.com/publiccodeyml/publiccode.yml?
+
+## Protocol
+
+* Onboarding liblrs, we will do next week, when maintainer is back
+* DCO bot for TC repo switched on
+  * It works in the web interface automatically when the setting is enabled for the repo
+  * In git you have to add the -s option to sign-off commits, automation is not possible by default as it is supposed to be an explicit statement by the developer
+* GitHub gave us teams plan for nonprofit organization
+* Managing OpenRail Association owned DNS entries: https://github.com/OpenRailAssociation/openrail-dns/
+  * Semi-automatic automation via the data in the repository
+  * Data is publicly available from DNS in principle anyway, so it is not a problem to have it in a public repo. There might be edge cases, where we could introduce GPG-signed commits to make sure that changes are only coming from the people they are supposed to come from, but might be not worth the effort right now.
+  * Similar as with the GitHub team management there could be issues if within the people who have legitimate access to the files are changing things from other projects. This is prevented by the CODEOWNERS and review mechanisms, but there are still some technical possibilities left. We rely on the social contract that this is not happening.
+  * Decision: We use the openrail-dns repo to manage the domains which are managed automatically as Max proposes.
+* Discussion: For describing projects, would it make sense to adopt something like https://github.com/publiccodeyml/publiccode.yml?
+  * It's a standard for specifying metadata for open source projects coming from public administration
```

Comment:
```suggestion
  * It's a standard for specifying metadata for open-source projects coming from public administration
```

Reply:
While this might be correct in terms of strict grammar rules, the spelling without hyphen also is very common (and as far as I understand not strictly incorrect). As we usually use it this way, I would let it stand here without hyphen as well.